### PR TITLE
fix complex parameters

### PR DESCRIPTION
**Pull request by vincentgoat**
Signed-off-by: vincentgoat <linguohui1@huawei.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
1. Functions always returning same value
2. The Quic and WS connections of the Viaduct package contain logic that could be removed altogether. In both cases, the switch statement can be removed. A comment should be left to explain why the functions look like they do. It could be considered whether `processControlMessage` could be removed entirely.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: edge/pkg/devicetwin/dtmanager/membership.go [hunk 1]
```diff
@@ -345,11 +347,13 @@ func dealMembershipGetInner(context *dtcontext.DTContext, payload []byte) error
 	topic := dtcommon.MemETPrefix + context.NodeName + dtcommon.MemETGetResultSuffix
 	klog.Infof("Deal getting membership successful and send the result")
 
-	context.Send("",
+	err = context.Send("",
 		dtcommon.SendToEdge,
 		dtcommon.CommModule,
 		context.BuildModelMessage(modules.BusGroup, "", topic, messagepkg.OperationPublish, result))
-
+	if err != nil {
```
Comment (wackxu) [inline edge/pkg/devicetwin/dtmanager/membership.go:354]: suggest  `return err`

File: edge/pkg/edged/volume/csi/csi_plugin.go [hunk 2]
```diff
@@ -126,7 +124,7 @@ func (h *RegistrationHandler) ValidatePlugin(pluginName string, endpoint string,
 func (h *RegistrationHandler) RegisterPlugin(pluginName string, endpoint string, versions []string) error {
 	klog.Infof(log("Register new plugin with name: %s at endpoint: %s", pluginName, endpoint))
 
-	highestSupportedVersion, err := h.validateVersions("RegisterPlugin", pluginName, endpoint, versions)
```
Comment (gy95) [inline edge/pkg/edged/volume/csi/csi_plugin.go:129]: this file is derived from kubernetes, is this need fixing?

**Review by gy95 — APPROVED** [review]
/lgtm

**Review by fisherxu — COMMENTED** [review]
/approve

**Pull request merged**